recipes/meshpy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "meshpy" %}
+{% set uname = "MeshPy" %}
+{% set version = "2016.1.2" %}
+{% set sha256 = "43ff2f9b010183ed679632a6eaa3ff8fb7a2015024aee5cbb00ea49c821ac0b4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ uname }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ uname[0] }}/{{ uname }}/{{ uname }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt && cd test && python -m pytest
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - pytest
+    - six
+  run:
+    - python
+    - setuptools
+    - pytools
+    - numpy
+    - six
+
+test:
+  requires:
+    - pytest
+  imports:
+    - meshpy
+
+about:
+  home: http://github.com/inducer/meshpy
+  license: MIT, Proprietary (Triangle, included), GNU Affero General Public License (TetGen, included)
+  license_file: doc/faq.rst
```

Comment:
@msarahan any advice on what to do with multiple licenses? I don't think the FAQ should go here. At least let's add the "main" license file from `meshpy`.